cmd_raw_topic: cmd_raw
cmd_str_topic: cmd_str
cmd_vel_topic: cmd_vel
